src/scene/graphics/shared/path/GraphicsPath.ts

```diff
@@ -363,6 +367,13 @@ export class GraphicsPath
         return this;
     }
 
+    public invalidate()
```

Comment:
yoo, I think we can delete this function?

Reply:
Absolutely!